MAINTAINERS.md

```diff
@@ -143,6 +146,17 @@ git push origin <tagname>
 - [ ] Merge `master` back into `develop`.
 - [ ] Cut a new release branch from `develop`.  The branch should have the format `release/urbit-os-vX.XX`
 
+### Release Next Release Candidate
+
+We'll now need to **IMMEDIATELY** deploy the new release candidate to the pre-release moons. 
```

Comment:
Might be worth explicitly saying before anything new merges into develop after branching for release (the why)?

Reply:
Deploying directly from release branches to marnec via tooling will fix this.